pkg/daemon/ceph/client/image.go

```diff
@@ -63,6 +64,24 @@ func ListImages(context *clusterd.Context, clusterName, poolName string) ([]Ceph
 	return images, nil
 }
 
+func getImageInfo(context *clusterd.Context, clusterName, name, poolName string) (*CephBlockImage, error) {
+	imageSpec := getImageSpec(name, poolName)
+	args := []string{"info", imageSpec}
+	buf, err := ExecuteRBDCommand(context, clusterName, args)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get image %s info: %+v", imageSpec, err)
+	}
+
+	var image CephBlockImage
+	if err = json.Unmarshal(buf, &image); err != nil {
```

Comment:
Still have to account for `ROOK_LOG_LEVEL=DEBUG` - `librados` debug messages mixing with `rbd info`'s json output.

Reply:
Solved in https://github.com/rook/rook/pull/2257/commits/a3c00adffcd660f61c4bc5cac85ef414d1eaa648 .

Reply:
Reverted, as the debugging output issue was already fixed in #2247 .